src/dialog_ffi.mjs

```diff
@@ -0,0 +1,11 @@
+export function showModal(element) {
+    if (!element || !(element instanceof HTMLDialogElement)) return;
```

Comment:
I would really rather not have an API that silently fails.

This should return an error, if the Gleam function accepts any Element

Also if this is an Element Type (from Gleam viewpoint) it should be in Element file/module